shuup/xtheme/templates/shuup/xtheme/editor.jinja

```diff
@@ -11,7 +11,7 @@
     <script type="text/javascript" src="{{ url("shuup_admin:js-catalog") }}"></script>
     <script src="{{ static("xtheme/editor.js") }}"></script>
     <link href="https://netdna.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.css" rel="stylesheet">
-    <link href="https://cdnjs.cloudflare.com/ajax/libs/summernote/0.8.2/summernote.css" rel="stylesheet">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/summernote/0.8.10/summernote.css" rel="stylesheet">
```

Comment:
I think we should start serving these as a part of the Shuup installation. I will create a separate issue about this as well, but could you please include and build this CSS file locally instead of the CDN.

Reply:
OK, I guess this could be done after https://github.com/shuup/shuup/pull/1335 gets merged and be part of https://github.com/shuup/shuup/issues/1404 as changing only this won't be super effective now.